<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="checkBoxLogPlot">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>770</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Log Scale</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxModel1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>561</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Model 1</string>
   </property>
   <widget class="QPushButton" name="pushButtonFormula_1">
    <property name="geometry">
     <rect>
      <x>464</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Formula</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam1_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam2_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam1_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam2_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>delta^2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRelLikelihood_1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBIC_1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelRelLikelihood_1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rel. Likelihood</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBIC_1">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BIC</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonClipboard">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>370</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Results to Clipboard</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxModel2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>561</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Model 2</string>
   </property>
   <widget class="QPushButton" name="pushButtonFormula_2">
    <property name="geometry">
     <rect>
      <x>464</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Formula</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam1_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam2_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam1_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam2_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>delta^2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRelLikelihood_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBIC_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelRelLikelihood_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rel. Likelihood</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBIC_2">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BIC</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParam3_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelParam3_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>alpha</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonAnalyze">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Analyze</string>
   </property>
  </widget>
  <widget class="ModelFitWidget" name="plot">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>561</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ModelFitWidget</class>
   <extends>QGraphicsView</extends>
   <header>gui.tab.msd</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
